os/board/rtl8721csm/src/component/soc/realtek/amebad/fwlib/ram_hp/rtl8721dhp_app_start.c

```diff
@@ -41,22 +41,7 @@ extern void vPortExitCritical(void);
 extern unsigned int _ebss;
 extern unsigned int __StackLimit;
 extern unsigned int __PsramStackLimit;
-#define IDLE_STACK ((uintptr_t)&_ebss+CONFIG_IDLETHREAD_STACKSIZE-4)
-#define HEAP_BASE  ((uintptr_t)&_ebss+CONFIG_IDLETHREAD_STACKSIZE)
-#define HEAP_LIMIT ((uintptr_t)&__StackLimit)
-#define PSRAM_HEAP_BASE ((uintptr_t)&__psram_bss_end__[0])
-#define PSRAM_HEAP_LIMIT ((uintptr_t)&__PsramStackLimit)
-void os_heap_init(void){
-	kregionx_start[0] = (void *)HEAP_BASE;
-	kregionx_size[0] = (size_t)(HEAP_LIMIT - HEAP_BASE);
-	if(TRUE == psram_dev_config.psram_dev_enable) {
-		kregionx_start[1] = (void *)PSRAM_HEAP_BASE;
-		kregionx_size[1] = (size_t)(PSRAM_HEAP_LIMIT - PSRAM_HEAP_BASE);
```

Comment:
Could you confirm that `PSRAM_HEAP_BASE` and size are the same as our configurations?
It means there is no data in psram text, data and bss sections.

Reply:
Yes, I have verified PARAM_HEAP_BASE and size are same as the one calculated at up_allocateheap.c.
There is no text, data, bss in psram.